OpenRA.Mods.Common/SpriteLoaders/ShpTSLoader.cs

```diff
@@ -118,7 +118,9 @@ bool IsShpTS(Stream s)
 			while (w == 0 && h == 0 && f++ < imageCount);
 
 			s.Position = start;
-			return type < 4;
+
+			// Type 18 is a special case for RA2 low bridges.
```

Comment:
I'm sorry to say that i'm pretty sure that this is not a special case, but the result of some other bug that still hasn't been found and needs to be fixed.

https://github.com/zzattack/ccmaps-net/blob/master/CNCMaps.FileFormats/ShpFile.cs#L58 does not have any special handing for a type 18, and that project has no issues with the RA2 lobrdb sprites.